### ISSUE TITLE
Use Ruff for formatting?

### ISSUE THREAD

**Issue by cbrnr** [the-siesta-group/edfio#32]
Ruff has gained a Black-style formatting option, which means that formatting is a lot faster (just like linting). Do you want to switch from Black to Ruff? I've done this e.g. for SleepECG here: https://github.com/cbrnr/sleepecg/pull/204

**Comment by hofaflo** [comment]
Yes, definitely! Though we'd like to wait until the 2024 style is stable (and maybe until the formatter leaves beta) to avoid multiple small changes regarding that.
